AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/holder/HasMethodInjection.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (C) 2010-2015 eBusiness Information, Excilys Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed To in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.androidannotations.holder;
+
+import javax.lang.model.element.Element;
+
+import com.helger.jcodemodel.IJExpression;
+import com.helger.jcodemodel.JBlock;
+
+public interface HasMethodInjection<T extends GeneratedClassHolder> {
```

Comment:
This is strange that this interface is in `holder` package but is implemented by `Handler`. 
`Holders` have injection methods, not `Handlers`. So this interface should be implemented by `Holders`.